spec/factories/messages.rb

```diff
@@ -38,4 +38,22 @@
       )
     end
   end
+
+  trait :with_docx_attachment do
+    attachment do
+      Rack::Test::UploadedFile.new(
+        File.expand_path('features/examples/shorter_lorem.docx', Rails.root),
+        'application/msword'
+      )
+    end
+  end
+
+  trait :with_pdf_attachment do
+    attachment do
+      Rack::Test::UploadedFile.new(
+        File.expand_path('features/examples/longer_lorem.pdf', Rails.root),
+        'application/pdf'
+      )
+    end
+  end
```

Comment:
do we need this if we use documents factory -  related to [comment](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/3852/files#r604083739)